recipes/ssh2-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ssh2-python" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "e5e77801a7a0d49799e288ecbd189b4d1c23eeadcf9c6f4ce80b31b7d397babe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
```

Comment:
This is only needed for things like C++ libraries where we need an explicit way to specify `vc`. The Python interpreter version already handles matching the `vc` runtime for us. So there is no need for Python packages to do this even if they include compiled portions.